firefox_desktop/firefox_desktop.model.lkml

```diff
@@ -1,6 +1,125 @@
 connection: "telemetry"
 
-include: "//looker-hub/firefox_desktop/views/*.view.lkml"
+include: "//looker-hub/firefox_desktop/views/active_users_aggregates_table.view.lkml"
+include: "//looker-hub/firefox_desktop/views/adclick_history_table.view.lkml"
```

Comment:
Adding all includes here for now. They are not all needed and should be trimmed down, but as part of this PR listing them all is identical to `include: "//looker-hub/firefox_desktop/views/*.view.lkml"` and ensures that no errors get introduced

Reply:
I think it'd be good to leave `# TODO` comments in these cases indicating that we should pare these view includes down to the minimum necessary, as the large number of them may be contributing toward longer LookML validation times.